index.html

```diff
@@ -230,6 +230,84 @@ <h1>Skills</h1>
           </ul>
         </div>
       </section>
+      <section class="contact">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/{form_id}" method="post">
+          <ul class="ContactForm">
+            <li>
+              <input
+                type="text"
+                name="name"
+                class="inputText"
+                placeholder="Full name"
+                required
+                maxlength="30"
+                type="text"
+              />
+            </li>
+            <li>
+              <input
+                class="inputText"
+                type="email"
+                name="email"
+                placeholder="Email address"
+                required
+                type="email"
+              />
+            </li>
+            <li>
+              <textarea
+                class="inputTextarea"
+                name="comment"
+                id="comment"
+                cols="50"
+                rows="10"
+                placeholder="Enter text here"
+                maxlength="500"
+                required
+                type="text"
+              ></textarea>
+            </li>
+            <li>
+              <button class="contact-button">Get In Touch</button>
```

Comment:
- [ ] Good job adding the hover state 👍🏽 . When I hover the button, I noticed that the color of the button text is white making the button not visible. Please make sure your button interaction matches the Figma Design 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/192594405-2053af26-76eb-464e-8e25-61c821682e4e.png)